build-manual

```diff
@@ -1,8 +1,13 @@
 #!/bin/sh
 
-python -m pip install --pre -r requirements.txt
+python -m pip install -r requirements.txt
+
+python -m pip install --pre git+git://github.com/willsimmons1465/jupyter-sphinx.git@issue-140#egg=jupyter-sphinx
 
 cd manual/
+
+sed "s/REQUIREMENTS/$(sed -e 's/[\&/]/\\&/g' -e 's/$/\\n/' ../requirements.txt | tr -d '\n')/" index-template.rst > index.rst
```

Comment:
Can we remove `index.rst` after the doc build?

Reply:
Yes we should...